install-local.sh

```diff
@@ -144,6 +167,28 @@ ns_timestamp=$(date +%Y-%m-%ST%H:%M:%S%z | sed 's/[0-9][0-9]$/:&/')
 echo "$ns_timestamp" > "$ns_build_path/timestamp"
 echo "${ns_sysname:=$(hostname -s)}" > "$ns_build_path/sysname"
 
+# Set up python virtual environment.
+
+if [ "$ns_pyvenv" != disable ]; then
+    echo
+    msg "Initializing python virtual environment"
+    ns_pyvenv_opt=
+    if [ "$ns_pyvenv" == inherit ]; then
+	ns_pyvenv_opt=--system-site-packages
+    fi
+
+    msg "Installing python modules: $ns_pyvenv_modules"
+    (
+	exec >> "$ns_build_path/log_pyvenv" 2>&1
+	if python3 -m venv $ns_pyvenv_opt "$ns_pyvenv_path"; then
```

Comment:
Should this be `${ns_python}`? `python3` might not be enough, e.g. `python3.7`.

Reply:
It should, thanks.